service/build.gradle.kts

```diff
@@ -67,6 +67,7 @@ dependencies {
   implementation(Dependencies.wispConfig)
   implementation(Dependencies.wispDeployment)
   implementation(Dependencies.wispLogging)
+  implementation(Dependencies.slf4jSimple)
```

Comment:
Not sure if this is a good idea.  Looks like something similar is happening here:
https://github.com/cashapp/backfila/pull/339

Reply:
This might be why the tests are failing.